docs/source/svi.rst

```diff
@@ -9,4 +9,15 @@ Stochastic Variational Inference (SVI)
 
 ELBo
 ----
-.. autofunction:: numpyro.infer.svi.elbo
+
```

Comment:
Not for this PR, but we should rename this `ELBO` too to match the rest of the codebase.

Reply:
done